packages/lib-classifier/src/components/Classifier/components/SubjectViewer/components/LightCurveViewer/LightCurveViewerContainer.spec.js

```diff
@@ -182,7 +187,7 @@ describe('Component > LightCurveViewerContainer', function () {
       )
     })
 
-    it('should update component state when there is a new valid subject', function (done) {
+    it.skip('should update component state when there is a new valid subject', function (done) {
```

Comment:
Same here, but I haven’t really dived into the LCV code to see what it’s doing differently from other viewers. 

That viewer is pretty much deprecated since [ADR  28](https://github.com/zooniverse/front-end-monorepo/blob/master/docs/arch/adr-28.md) so I’m not too fussed about this.